packages/modules/devices/rct/bat.py

```diff
@@ -27,16 +27,18 @@ def update(self, rct_client: RCT) -> None:
         # read all parameters
         rct_client.read(my_tab)
 
-        if (stat1.value + stat2.value + stat3.value) > 0:
-            raise FaultState.error("Alarm Status Speicher ist ungleich 0.")
-
         bat_state = BatState(
             power=watt1.value * -1,
             soc=socx.value * 100,
             imported=watt2.value,
             exported=watt3.value
         )
         self.store.set(bat_state)
+        if (stat1.value + stat2.value + stat3.value) > 0:
+            # Werte werden trotz Fehlercode übermittelt.
+            self.fault_state.warning(
```

Comment:
Das neue Error-Handling gibt es in 1.9 nicht. Hier müsste die Fehlermeldung dann mit log.warning geloggt werden.

Reply:
Ist angepasst.